koku/masu/database/trino_sql/openshift/cost_model/monthly_cost_gpu.sql

```diff
@@ -0,0 +1,89 @@
+INSERT INTO postgres.{{schema | sqlsafe}}.reporting_ocpusagelineitem_daily_summary (
+    uuid,
+    report_period_id,
+    cluster_id,
+    cluster_alias,
+    data_source,
+    usage_start,
+    usage_end,
+    namespace,
+    node,
+    resource_id,
+    pod_labels,
+    all_labels,
+    source_uuid,
+    cost_model_rate_type,
+    cost_model_gpu_cost,
+    monthly_cost_type,
+    cost_category_id
+)
+SELECT
+    uuid() as uuid,
+    {{report_period_id}} as report_period_id,
+    CAST({{cluster_id}} AS varchar(50)) as cluster_id,
+    CAST({{cluster_alias}} AS varchar(256)) as cluster_alias,
+    CAST('GPU' AS varchar(64)) as data_source,
+    date(gpu.interval_start) as usage_start,
+    date(gpu.interval_start) as usage_end,
+    CAST(gpu.namespace AS varchar(253)) as namespace,
+    CAST(gpu.node AS varchar(253)) as node,
+    CAST(gpu.gpu_uuid AS varchar(253)) as resource_id,
+    cast(map(
+        ARRAY['gpu-model', 'gpu-vendor', 'gpu-memory-mib', 'pod-name'],
+        ARRAY[
+            gpu.gpu_model_name,
+            gpu.gpu_vendor_name,
+            CAST(gpu.gpu_memory_capacity_mib AS varchar),
+            gpu.pod
+        ]
+    ) as json) as pod_labels,
+    cast(map(
```

Comment:
Lets just have this information in all-labels, and not worry about inserting into pod_labels.